channeldb/graph.go

```diff
@@ -638,8 +715,8 @@ func (c *ChannelGraph) ForEachNodeCacheable(cb func(kvdb.RTx,
 			}
 
 			nodeReader := bytes.NewReader(nodeBytes)
-			err := deserializeLightningNodeCacheable(
-				nodeReader, cacheableNode,
+			cacheableNode, err := deserializeLightningNodeCacheable(
```

Comment:
Rationale?

Reply:
We want to use `cacheableNode` outside of the callback. See also 
https://github.com/lightningnetwork/lnd/pull/6111#discussion_r779661615 and https://github.com/lightningnetwork/lnd/pull/6111#issuecomment-1010909520